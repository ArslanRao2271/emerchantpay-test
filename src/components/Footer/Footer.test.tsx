import { render, screen } from "@testing-library/react";
import { Footer } from ".";
import { MemoryRouter } from "react-router-dom";

const getWrapperComponent = (comp: JSX.Element) => {
  return render(comp, { wrapper: MemoryRouter });
};

describe("Footer test cases", () => {
  test("Should match previous snapshot", () => {
    const { asFragment } = getWrapperComponent(<Footer />);

    expect(asFragment()).toMatchSnapshot();
  });

  test("Should render footer text", () => {
    getWrapperComponent(<Footer />);

    expect(screen.queryByText(/E-MerchantPay/)).toBeVisible();
  });
});
