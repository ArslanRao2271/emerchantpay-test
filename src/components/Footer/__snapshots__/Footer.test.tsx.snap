// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer test cases Should match previous snapshot 1`] = `
<DocumentFragment>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-4zl1qb-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
    >
      <div
        class="MuiTypography-root MuiTypography-h6 css-1t4mx1v-MuiTypography-root"
      >
        E-MerchantPay
      </div>
    </div>
  </header>
</DocumentFragment>
`;
